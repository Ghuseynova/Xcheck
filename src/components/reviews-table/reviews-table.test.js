import React from 'react';
import { shallow } from 'enzyme';
import ReviewsTable from './reviews-table';

describe('render ', () => {
	it('Reviews Table', () => {
		const tree = shallow(<ReviewsTable/>);
		expect(tree).toMatchSnapshot();
	});
});