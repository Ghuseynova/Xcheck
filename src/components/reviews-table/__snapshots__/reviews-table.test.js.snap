// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render  Reviews Table 1`] = `ShallowWrapper {}`;
